core/domain/stats_services.py

```diff
@@ -334,11 +334,8 @@ def update_exp_issues_for_new_exp_version(
         playthrough_model.actions = [
             action.to_dict() for action in playthrough.actions]
 
-    # Run in transaction to help prevent data-races between operations that may
-    # update the playthroughs concurrently.
     stats_models.PlaythroughModel.update_timestamps_multi(playthrough_models)
-    transaction_services.run_in_transaction(
-        stats_models.PlaythroughModel.put_multi, playthrough_models)
+    stats_models.PlaythroughModel.put_multi(playthrough_models)
```

Comment:
Removes run_in_transaction for PlaythroughModel.put_multi method as it leads to issues on prod: BadRequestError: operating on too many entity groups in a single transaction.